pombola/south_africa/static/sass/write.scss

```diff
@@ -0,0 +1,1377 @@
+body {
+    color: #333;
+}
+
+.write-main {
+    .svg-icon {
+        display: -webkit-box;
+        display: -webkit-flex;
+        display: -ms-flexbox;
+        display: flex;
+        width: 1.2em;
+        height: 1.2em;
+        -webkit-box-pack: center;
+        -webkit-justify-content: center;
+        -ms-flex-pack: center;
+        justify-content: center;
+        -webkit-box-align: center;
+        -webkit-align-items: center;
+        -ms-flex-align: center;
+        align-items: center;
+        -webkit-box-flex: 0;
+        -webkit-flex: 0 0 auto;
+        -ms-flex: 0 0 auto;
+        flex: 0 0 auto;
+    }
+
+    .svg-icon.is--small {
+        width: 0.8em;
+        height: 0.8em;
+    }
+
+    .svg-icon.is--very-small {
+        width: 0.5em;
+        height: 0.5em;
+    }
+
+    .svg-icon.is--large {
+        width: 1.6em;
+        height: 1.6em;
+    }
+    .modal-dialog {
+        max-width: none;
+        overflow-y: initial;
+    }
+    .write-main__wrapper {
+        position: relative;
+        display: -webkit-box;
+        display: -webkit-flex;
+        display: -ms-flexbox;
+        display: flex;
```

Comment:
In future let's find and use a mixin for all the browser prefixes to avoid having to repeat them every time we want `display: flex`

Reply:
noted